custom/icds/models.py

```diff
@@ -92,6 +92,12 @@ class HostedCCZ(models.Model):
     profile_id = models.CharField(max_length=255, blank=True)
     file_name = models.CharField(max_length=255, blank=True)
     note = models.TextField(blank=True)
+    status = models.CharField(max_length=255, null=False, blank=False, default='pending',
+                              choices=(('pending', 'Pending'),
```

Comment:
Nitpick: constants are better than magic strings, both here and in `update_status` below.